docs/make.jl

```diff
@@ -42,104 +68,71 @@ function compute_GAP_URL(url::String)
   return String(urls[1][3])
 end
 
-## Distinguish different versions of Documenter.jl.
-if isdefined(Documenter, :Document)
-  # version at least 1.0
-  # The code was adapted from `expand_citations`
-  # in DocumenterCitations.jl/src/citations.jl.
-  Document = Documenter.Document
-
-  # Execute our step before the computation of cross-references (3.0)
-  # because otherwise the syntax of external references leads to errors.
-  Documenter.Selectors.order(::Type{ExternalReference}) = 2.99
-
-  function compute_external_references(doc::Document)
-    for (src, page) in doc.blueprint.pages
-      @debug "ExternalReference: compute for entries in $(src)"
-      empty!(page.globals.meta)
-      compute_external_reference(doc, page, page.mdast)
-    end
-  end
 
-  function compute_external_reference(doc::Document, page, mdast::Documenter.MarkdownAST.Node)
-    for node in Documenter.AbstractTrees.PreOrderDFS(mdast)
-      if node.element isa Documenter.DocsNode
-        # The docstring AST trees are not part of the tree of the page, so
-        # we need to expand them explicitly
-        for (docstr, meta) in zip(node.element.mdasts, node.element.metas)
-          compute_external_reference(doc, page, docstr)
-        end
-      elseif node.element isa Documenter.MarkdownAST.Link
-        compute_external_reference(node, page.globals.meta, page, doc)
-      end
-    end
-  end
+# The code was adapted from `expand_citations`
+# in DocumenterCitations.jl/src/citations.jl.
```

Comment:
There is no longer a file `src/citations.jl`, and there is no function `expand_citations` anymore in DocumenterCitations.jl.

Reply:
This is part of existing code that was just moved around (and that bit was in another PR and thus is already merged). I believe you wrote it, @ThomasBreuer ? If this is outdated, can you please open a PR updating it?

Reply:
Already did that in [`3326f7b` (#1130)](https://github.com/oscar-system/GAP.jl/pull/1130/commits/3326f7b7ac3412829bda74d0aac30d62ce8ad193)

Reply:
(Sorry, this was a misunderstanding:
Yes, I had added this comment when I added the code. With the current changes of the code, the comment was no longer relevant; this would hold even if the claimed reference would still exist.)